README.md

```diff
@@ -95,6 +95,20 @@ $ docker-compose up
 
 Agora basta acessar o navegador em [localhost:8000](http://localhost:8000) para ver o conteúdo gerado.
 
+Instalando e Rodando no Windows (Docker)
+--------------------------
+- Instale o [Visual Studio Code](https://code.visualstudio.com/) para fazer códigos legais;
```

Comment:
tem como não usar o Visual Studio Code ou é obrigatório?

se não for obrigatório acho que vale a pena marcar como opcional

Reply:
Done!